Intersect.Server/Networking/PacketHandler.cs

```diff
@@ -212,6 +212,109 @@ public bool HandlePacket(IConnection connection, IPacket packet)
                 return false;
             }
 
+            if (packet is AbstractTimedPacket timedPacket)
+            {
+                var ping = connection.Statistics.Ping;
+                var ncPing = (long) Math.Ceiling(
+                    (connection as LidgrenConnection).NetConnection.AverageRoundtripTime * 1000
+                );
+
+                var localAdjusted = Timing.Global.Ticks;
+                var localAdjustedMs = localAdjusted / TimeSpan.TicksPerMillisecond;
+                var localOffsetMs = Timing.Global.MillisecondsOffset;
+                var localUtcMs = localOffsetMs + localAdjustedMs;
+
+                var remoteAdjusted = timedPacket.Adjusted;
+                var remoteAdjustedMs = remoteAdjusted / TimeSpan.TicksPerMillisecond;
+                var remoteUtcMs = timedPacket.UTC / TimeSpan.TicksPerMillisecond;
+                var remoteOffsetMs = timedPacket.Offset / TimeSpan.TicksPerMillisecond;
+
+                var deltaAdjusted = localAdjustedMs - remoteAdjustedMs;
+                var deltaWithPing = deltaAdjusted - ping;
```

Comment:
Would this not be deltaWithoutPing ?

Reply:
I think it reads deltaWithPing because it is delta taking ping into account. Idk. @lodicolo?